Rewards/Classes/Account/AccountProvider.swift

```diff
@@ -4,9 +4,12 @@
  */
 
 import Foundation
+import CaptureReceipt
```

Comment:
is it used?

Reply:
Yes, we uso in the Email and Retailer Enum